### ISSUE TITLE
Abou code error for online mapping test

### ISSUE THREAD

**Issue by 1Hun0ter1** [yuantianyuan01/PreSight#4]
I think that codes for online mapping test using by streammapnet have many bugs, please you give us corrected codes.

**Comment by yuantianyuan01** [comment]
I understand your frustration, but in order to help resolve the issue, I need more detailed information. When reporting bugs, it's essential to include details such as your environment (e.g., conda), the hardware you're using (GPUs), and the specific error or line where the issue occurs. This helps us investigate and address any problems more effectively. We appreciate your understanding and look forward to your detailed report.

**Comment by 1Hun0ter1** [comment]
Thanks for your reply, I am using mmcv-full=1.5.3 but not mmcv=1.7.0. In this condition, I can also successfully test after updating the published codes. But training and testing code should be different.

**Comment by 1Hun0ter1** [comment]
> I understand your frustration, but in order to help resolve the issue, I need more detailed information. When reporting bugs, it's essential to include details such as your environment (e.g., conda), the hardware you're using (GPUs), and the specific error or line where the issue occurs. This helps us investigate and address any problems more effectively. We appreciate your understanding and look forward to your detailed report.

When I installed mmcv=1.7.0 and then to test through `sh tools/dist_test.sh ./plugin/configs/smn_priorsplit_480_100x50_24e.py /data.pth 8,` an error arose: No module named 'mmcv._ext'. How to solve it? I switched from mmcv to mmcv-full, is it ok?

**Comment by yuantianyuan01** [comment]
Yes, you should try installing mmcv-full and that should fix the issue. Just make sure you followed their official instructions.

**Comment by 1Hun0ter1** [comment]
> Yes, you should try installing mmcv-full and that should fix the issue. Just make sure you followed their official instructions.

In this way, I should revise many places in the current codes.
